src/modules/Completelist.js

```diff
@@ -17,7 +17,7 @@ class Completelist {
     const clearButton = document.querySelector('.clear');
     clearButton.addEventListener('click', () => {
       let todolists = LocalStorage.getToDoLists();
-      todolists = todolists.filter((elemnt) => elemnt.comp !== true);
+      todolists = todolists.filter((elemnt) => elemnt.completed !== true);
       localStorage.setItem('todolists', JSON.stringify(todolists));
       window.location.reload();
```

Comment:
- [Optional] You did great adding the changes as requested @yusufyusie. But, I notice your task remains an input field after adding the task. It is not a good Ides, It should be turned into the input field usually when the user clicks on the edit task. for the time being, I would suggest adding the task as `<p>` or something else other than the input field.